### PR TITLE
added seed data

### PR DESCRIPTION
**Pull request by vinnyalfieri**
Added seed data.

close issue #1 

@ipc103 @jmburges @deniznida

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    "Ada Lovelace" => {
+      :year_completed => 1852
+    },
+    "Linus Torvalds" => {
+      :year_completed => 1969
+    }
+  }
+
+figure_list.each do |name, figure_hash|
+  p = Figure.new
```
Comment (jmburges) [inline db/seeds.rb:52]: Why do you have the `figure_hash` stuff without using it?

**Comment by vinnyalfieri** [comment]
@jmburges It does not add any value. Do you want me to remove it?

**Comment by jmburges** [comment]
@vinnyalfieri yeah if it doesn't use that data, then there is no reason to add it to the hash

**Comment by jmburges** [comment]
Maybe just an array of names?

**Comment by vinnyalfieri** [comment]
@jmburges @AnnJohn 
Updated seed data to remove unused elements. This seed data corresponds to the other PR for solution branch.

**Pull request merged**